apps/web/src/components/settings/ProfileSettings.tsx

```diff
@@ -5,50 +5,74 @@ import { Button } from "@/components/shadcn/ui/button";
 import { Label } from "@/components/shadcn/ui/label";
 import { Textarea } from "@/components/shadcn/ui/textarea";
 import {
-	modifyRegistrationData,
+	modifyProfileData,
 	updateProfileImage,
 } from "@/actions/user-profile-mod";
 import { useUser } from "@clerk/nextjs";
 import { useAction } from "next-safe-action/hook";
 import { toast } from "sonner";
 import { useState } from "react";
 import { encodeFileAsBase64 } from "@/lib/utils/shared/files";
+import { Tag, TagInput } from "@/components/shadcn/ui/tag/tag-input";
+import { Loader2 } from "lucide-react";
+import { Avatar, AvatarImage } from "@/components/shadcn/ui/avatar";
 
-interface ProfileSettingsProps {
+interface ProfileData {
+	pronouns: string;
 	bio: string;
-	university: string;
+	skills: string[];
+	discordUsername: string;
+	profilePhoto: string;
 }
 
-export default function ProfileSettings({
-	bio,
-	university,
-}: ProfileSettingsProps) {
-	const [newBio, setNewBio] = useState(bio);
-	const [newUniversity, setNewUniversity] = useState(university);
+interface ProfileSettingsProps {
+	profile: ProfileData;
```

Comment:
question: Do you plan to add more to these props? If it is just one attribute, it can live on its own.

Reply:
As of right now, no. I just copied it this way bc it was the format of when i got it. I can change it to be on its own.